editor/editor_plugin.cpp

```diff
@@ -662,7 +662,7 @@ void EditorPlugin::make_visible(bool p_visible) {
 }
 
 void EditorPlugin::edit(Object *p_object) {
-	if (p_object->is_class("Resource")) {
+	if (Object::cast_to<Resource>(p_object)) {
```

Comment:
Very minor: Not sure how expensive a cast is, but you can actually assign this as variable to prevent the second cast below.

Reply:
It's cheaper than the previous `is_class()` call.